functors_task_1/student/include/impl.h

```diff
@@ -16,16 +31,35 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand brand)
 {
+    switch (brand)
+    {
+        case BeerBrand::Corona:
+        case BeerBrand::HoeGaarden:
+        {
+            return true;
+        }
+        case BeerBrand::Leffe:
```

Comment:
Why do you need to mention all of these types?

Reply:
corrected